apps/response/sbox-intsvc/base/bitnami-helmrepo.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: HelmRepository
+metadata:
+  name: sbox-response-postgresql
+  namespace: flux-system
+spec:
+  url: https://charts.bitnami.com/bitnami/postgresql
+  interval: 10m
```

Comment:
This would be better placed inside the flux-system folder https://github.com/hmcts/cnp-flux-config/tree/master/apps/flux-system/base. It can just be called bitnami or bitnami-charts or something

Reply:
You could only install it on the sbox cluster by referencing it in this kustomization https://github.com/hmcts/cnp-flux-config/blob/master/apps/flux-system/sbox-intsvc/base/kustomization.yaml instead of the one where the other repos are setup https://github.com/hmcts/cnp-flux-config/blob/master/apps/flux-system/base/kustomization.yaml